presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoConfigDeployer.java

```diff
@@ -78,25 +83,34 @@ public void deployConfigFiles(File configTar, Path configDir)
             throw new PrestoManagerException(format("Failed to delete contents of the directory: %s", configDir.toString()), e);
         }
         int untarResult = executor
-                .runCommand("tar", "-xzvf", configTar.getAbsolutePath(),
+                .runCommand("tar", "-xzvf", configTar.toAbsolutePath().toString(),
                         "-C", configDir.toString());
         if (untarResult != 0) {
             throw new PrestoManagerException("Failed to deploy config files", untarResult);
         }
     }
 
-    public void addConfigFiles(Path configDir, Path dataDir, Path pluginDir)
+    /**
+     * Add default Presto configuration in the given directories
+     * <p>
+     * If {@link #defaultConfig} exists, its contents will be used for
+     * the configuration directory. Otherwise, non-configurable default
+     * configuration files will be created.
+     */
+    public void writeConfigFiles(Path configDir, Path dataDir, Path pluginDir)
```

Comment:
I believe that `add` is more misleading, because if the directory already exists, the first thing it does is _delete_ its contents; a function called `add` shouldn't be removing files.